src/scripts/love.js

```diff
@@ -0,0 +1,51 @@
+//  Description:
+//    For the giving of praise and thanks to colleagues.
+//    Looks for messages like "love @username for [reason]", acknowledges them,
+//    and copies them to #love.
+//
+//  Dependencies:
+//    None
+//
+//  Configuration:
+//    None
+//
+//  Commands:
+//    None
+
+const {
+  slack: { getChannelID },
+} = require("../utils");
+
+module.exports = (app) => {
+  app.message(
+    /^\s*(love|<3|:heart\w*:)\s+((<@[\w-]+>\s*)+)(.*)$/i,
+    async ({
+      client,
+      context: { matches },
+      event: { channel, ts: message, thread_ts: thread, user },
+      say,
+    }) => {
+      const lovees = matches[2].trim();
+
+      const { permalink } = await client.chat.getPermalink({
+        channel,
+        message_ts: message,
+      });
+
+      const channelID = await getChannelID("love");
+
+      client.chat.postMessage({
+        channel: "love",
+        icon_emoji: ":heart:",
+        text: `<@${user}> loves ${lovees}! <${permalink}|link>`,
```

Comment:
Only `channelID` could be undefined. `lovees` is guaranteed by the regex in the event subscription, `permalink` and `user` are guaranteed by the Slack API (because we got the inputs from Slack).

It won't fail, but it'll display something like, `Yay, more love for <PRIVATE CHANNEL>! Thanks, @whoever!`. Probably wouldn't hurt to put a check in for that.